downstream/modules/automation-hub/proc-configure-automation-hub-server-gui.adoc

```diff
@@ -0,0 +1,51 @@
+[id="proc-configure-automation-hub-server-gui"]
+== Configuring Red Hat {HubName} as the primary source for content using the web console
+
+Configure Red Hat automation hub as your primary source of content by using the web console. To configure automation hub, you must create a credential and add it to the Organization’s Galaxy Credentials field. With automation hub, you have access to certified, supported collections.
+
+.Prerequisites
+
+* You have obtained the API token for the {HubName} server. See link: https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/1.2/html/getting_started_with_red_hat_ansible_automation_hub/proc-create-api-token[Creating the {HubName} API token] for more information.
+[IMPORTANT]
+====
+	Creating a new token revokes any previous tokens 
+	generated for Private Automation Hub. Ensure that you update any Controller or scripts that you created with the previous token.
+====
+
+.Procedure
+
+. Navigate to your Automation Controller. 
+. Create a new credential.
+.. Click btn:[Add] from the **Credentials** screen.
+.. Enter the name for your new credential in the **Name** field.
+.. Optional: enter a description and enter or select the name of the organization with which the credential is associated.
+.. Under Organization, select the organization that you would like to use the credential for.
```

Comment:
Should 'Organization' and 'Galaxy Credentials' be in bold formatting, like the rest of the UI elements that are starting with 'Under ...' ? Please check the UI elements guidelines in the IBM Style Guide (pg. 215) and the screen and make the necessary formatting changes.